transaction/tests/StronglyConnectedComponents_unittest.cpp

```diff
@@ -43,7 +43,7 @@ class GraphConfiguration {
       std::unique_ptr<transaction_id> tid =
         std::make_unique<transaction_id>(transaction_id(index));
```

Comment:
We should refactor to

```
  const transaction_id transaction = static_cast<transaction_id>(index);
```

Also, rename `no_transaction` to `num_transactions`.